src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -256,6 +256,10 @@ Please note that the `<index>` denotes the number/index (in a form of an
 integer) of the echo not the echo time value which needs to be stored in the
 field EchoTime of the separate JSON file.
 
+The OPTIONAL `proc-<label>` key/value can be used to distinguish
+between original and processed on the scanner data, e.g. `_proc-norm`
+could be used for ["Normalized Pixel" field-bias corrected](https://github.com/nipy/heudiconv/issues/266#issuecomment-432662723) data.
+
```

Comment:
doh -- just now spotted that added it a bit too late -- will move it right after initial mentioning in `anat/`